tests/functional/aws-node-sdk/test/bucket/put.js

```diff
@@ -194,15 +194,35 @@ describe('PUT Bucket - AWS.S3.createBucket', () => {
         });
 
         describe('bucket creation success with object lock', () => {
-            function _testObjectLock(name, done) {
+            function _testObjectLockEnabled(name, done) {
                 bucketUtil.s3.createBucket({
                     Bucket: name,
                     ObjectLockEnabledForBucket: true,
+                }, (err, res) => {
+                    assert.ifError(err);
+                    assert(res.Location, 'No Location in response');
+                    assert.strictEqual(res.Location, `/${name}`,
```

Comment:
First assert is redundant as it can not be falsy and also strictly equal.
```suggestion
                    assert.strictEqual(res.Location, `/${name}`,
```